src/ussd_popi_rapidpro.js

```diff
@@ -927,37 +927,195 @@ go.app = function() {
+                next: "state_start",
+                text: $(
+                    "Thanks! MomConnect will continue to send you helpful messages." +
+                    " " +
+                    "Have a lovely day!"
+                )
+            });
+        });
+
+        self.add("state_anonymous_data_optout", function(name, opts){
+            var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+            var answers = self.im.user.answers;
+            var opt_out = answers.state_stop_being_part === "yes";
+            return self.rapidpro
+                .start_flow(
+                    self.im.config.optout_flow_id, null, "whatsapp:" + _.trim(msisdn, "+"), {
+                        opt_out: opt_out ? "TRUE" : "FALSE",
+                        mqr_consent: "Denied",
+                        research_consent: "FALSE"
+                    }
+                )
+                .then(function() {
+                    return self.states.create("state_opt_out_reason");
+                }).catch(function(e) {
+                    console.log(e.message);
+                    // Go to error state after 3 failed HTTP requests
+                    opts.http_error_count = _.get(opts, "http_error_count", 0) + 1;
+                    if(opts.http_error_count === 3) {
+                        self.im.log.error(e.message);
+                        return self.states.create("__error__", {return_state: name});
+                    }
+                    return self.states.create(name, opts);
+                });
+        });
+
+        self.add("state_stop_being_part_optout", function(name, opts){
+            var msisdn = utils.normalize_msisdn(self.im.user.addr, "ZA");
+            return self.rapidpro
+                .start_flow(
+                    self.im.config.clear_pii_flow, null, "whatsapp:" + _.trim(msisdn, "+")
+                )
+                .then(function() {
+                    return self.states.create("state_stop_being_part_optout_success");
+                }).catch(function(e) {
+                    // Go to error state after 3 failed HTTP requests
+                    opts.http_error_count = _.get(opts, "http_error_count", 0) + 1;
+                    if(opts.http_error_count === 3) {
+                        self.im.log.error(e.message);
+                        return self.states.create("__error__", {return_state: name});
+                    }
+                    return self.states.create(name, opts);
+                });
+
+        });
+
+        self.add("state_stop_being_part_optout_success", function(name) {
+            return new EndState(name, {
+                next: "state_start",
+                text: $(
+                    "Your research consent has been withdrawn, and you have been removed from all research." +
+                    " " +
+                    "MomConnect will continue to send you helpful messages." +
+                    " " +
+                    "Goodbye."
```

Comment:
This could be:
(
          "Your research consent has been withdrawn, and you have been removed from all research. " +
          "MomConnect will continue to send you helpful messages. Goodbye "
        ),        
OR 
You could use \n or join("\n")) if you want new lines.